src/UsgsAstroLsStateData.cpp

```diff
@@ -214,7 +218,24 @@ std::string UsgsAstroLsStateData::toString() const
    state_stream << STATE_KEYWORD[STA_PLATFORM_FLAG]     << " " << m_PlatformFlag      << "\n";  // 9
    state_stream << STATE_KEYWORD[STA_ABERR_FLAG] << " " << m_AberrFlag         << "\n";  // 10
    state_stream << STATE_KEYWORD[STA_ATMREF_FLAG]<< " " << m_AtmRefFlag        << "\n";  // 11
-   state_stream << STATE_KEYWORD[STA_INT_TIME]   << " " << m_IntTime           << "\n";  // 12
+   state_stream << STATE_KEYWORD[STA_INT_TIME_LINES]    << " " ;

```

Comment:
In terms of maintainability, I think we need to decide if serialization to JSON is sufficient and we can avoid also maintaining serialization `toString()`?